laserstudio/widgets/toolbars/lasertoolbar.py

```diff
@@ -2,4 +2,10 @@
 
 
 class LaserToolbar(QToolBar):
-    pass
+    def __init__(self, title: str, object_name: str):
```

Comment:
I would keep laserstudio as parent qt object through __init__ function accross LaserToolbar, LaserDriverToolbar and PDMToolbar, and also similar signature.

```suggestion
class LaserToolbar(QToolBar):
    def __init__(self, title: str, laser_studio: "LaserStudio", laser_num: int):
        super().__init__(title, laser_studio)
        super().setObjectName(f"toolbar-laser-{laser_num}")
```

And for subclasses:

```suggestion
class PDMToolbar(LaserToolbar):
    def __init__(self, title: str, laser_studio: "LaserStudio", laser_num: int):
        super().__init__(f"Laser {laser_num} (PDM)", laser_studio)
        super().setObjectName(f"toolbar-laser-pdm-{laser_num}")
```

Reply:
https://github.com/Ledger-Donjon/laserstudio/blob/920f22359cd3734ceaa29c9d59314ca2e31970e8/laserstudio/laserstudio.py#L108

Adding the toolbar already sets the parent of the widget, so I think it is not necessary to give explicitely the parent widget.

Reply:
Regarding the `laser_num` parameter, I woud still need to pass the `object_name` because both `LaserDriverToolbar` and `PDMToolbar` inherits `LaserToolBar` and we want to reflect this in the object name.

So maybe something like : `super().setObjectName(f"toolbar-laser-{model}-{laser_num}")`?

Reply:
Ok for this